kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -26,7 +26,6 @@
            the WAR depends on private modules. -->
       <!-- Keep the alphabetical order! -->
       <module name="org.apache.xerces"/>
-      <module name="org.apache.cxf"/>
```

Comment:
and this is the actual fix, remove the CXF from modules and rely on the one shipped with kie server